libs/core/src/lib/alert/alert.component.scss

```diff
@@ -2,6 +2,7 @@
 
 .fd-alert {
     display: block;
+    line-height: 1rem;
```

Comment:
This should be coming from Fundamental-styles

Reply:
Your right, it needed a class `fd-alert__text` wrapper.